src/Job/Import.php

```diff
@@ -2,9 +2,9 @@
 namespace FedoraConnector\Job;
 
 use Omeka\Job\AbstractJob;
-use EasyRdf_Graph;
-use EasyRdf_Resource;
-use EasyRdf_Namespace;
+use \EasyRdf\Graph;
```

Comment:
Style dictates no leading backslashes on `use` statements: they never respect the current namespace, so the leading backslash is unnecessary.